Domain.FSharp/Shipping/Cargo.fs

```diff
@@ -0,0 +1,14 @@
+namespace FSharp.Domain.Shipping.Cargo
+
+open System
+open System.Text.RegularExpressions
+
+type TrackingId(value: Guid) =
```

Comment:
Style nit: In F# everything is a tuple, applied as a function argument. Even for type declarations, adding a space can make the overall style more consistent (see similar style nits below).

```suggestion
type TrackingId (value: Guid) =
```

Reply:
That's what styling Rider gives me. I'll try to change it, but I'm not promising anything.